<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Genbank Processing by vinsondoyle</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Genbank Processing</h1>
      <h2 class="project-tagline"></h2>
      <a href="https://github.com/vinsondoyle/genbankProcessing" class="btn">View on GitHub</a>
      <a href="https://github.com/vinsondoyle/genbankProcessing/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/vinsondoyle/genbankProcessing/tarball/master" class="btn">Download .tar.gz</a>
    </section>

    <section class="main-content">
      <h1>
<a id="genbank-processing" class="anchor" href="#genbank-processing" aria-hidden="true"><span class="octicon octicon-link"></span></a>Genbank Processing</h1>

<p>Many phylogenetic analyses build on previous work published by others. Sometimes we use data from this work that is held in repositories such as <a href="http://datadryad.org/">Dryad</a> or <a href="treebase.org">TreeBase</a>, but there are also times when we want the untrimmed and unaligned sequences and thus download them from sequence databases such as <a href="http://www.ncbi.nlm.nih.gov/genbank/">GenBank</a>. Authors are required, hopefully, to report the GenBank accession numbers somewhere in the manuscript. We can use those accession numbers, such as <a href="http://www.ncbi.nlm.nih.gov/nuccore/JX145197.1">JX145197</a>, to find the referenced sequences and view them in several formats (fasta, genbank [.gb], asn). Fasta format is the most directly usable for phylogenetic studies because alignment programs will take a multi-fasta file (single file containing multiple fasta formatted sequences) and align the bases as an assessment of homology. However, the metadata associated with these sequences is lost once you convert them to fasta format, such as the journal, authors, and gene features. You can always look them up again, but I find that it is convenient to have these files available in the directory where I keep my data files for a particular project.</p>

<p>The scripts in this repository are meant to facilitate conversion from GenBank (.gb) flat file format to fasta and subsequently reformat the fasta files depending on your preferences. They are simple and probably duplicated elsewhere.</p>

<p>Read the usage info at the beginning of each script for instructions. </p>

      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/vinsondoyle/genbankProcessing">Genbank Processing</a> is maintained by <a href="https://github.com/vinsondoyle">vinsondoyle</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>

